cmd/res/devices/general.usb.camera.yaml.example

```
@@ -0,0 +1,9 @@
+DeviceList:
+  name: "example-camera"
+  profileName: "USB-Camera-General"
+  description: "Example Camera"
+  labels: [ "device-usb-camera-example", ]
+  Protocols:
+    USB:
+      path: "/dev/video0"
+      autoStreaming: "false"
```

These are map keys and should stay how they were in the TOML so they match the constants used in the code

```suggestion
      Path: "/dev/video0"
      AutoStreaming: "false"
```